packages/itwin/ec3-widget/src/components/api/context/EC3JobsClientContext.ts

```diff
@@ -0,0 +1,34 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import type { IEC3JobsClient } from "@itwin/insights-client";
+import { EC3JobsClient } from "@itwin/insights-client";
+import { createContext, useContext } from "react";
+
+export const BASE_PATH = "https://api.bentley.com/insights/carbon-calculation";
+
+const prefixUrl = (prefix: string) => {
+  return BASE_PATH.replace("api.bentley.com", `${prefix}-api.bentley.com`);
+};
```

Comment:
A lot of duplication. Also this doesn't seem right. If we get rid of context's this shouldn't be needed.